types/mocha/index.d.ts

```diff
@@ -1701,6 +1739,8 @@ declare namespace Mocha {
         private _onlyTests;
         private _onlySuites;
 
+        static constants: SuiteConstants;
```

Comment:
Should this be marked `readonly` or am I being paranoid?  (same for `Runner` constants)

Reply:
@cspotcode I made them readonly